src/Corporation/Corporation.tsx

```diff
@@ -277,10 +281,8 @@ export class Corporation {
     this.funds = this.funds - price;
 
     // Apply effects for one-time upgrades
-    if (upgN === 5) {
-      this.dividendTaxPercentage -= 5;
-    } else if (upgN === 6) {
-      this.dividendTaxPercentage -= 10;
+    if (upgN === 5 || upgN === 6) {
```

Comment:
nit: might not hurt to just always call it as the default seems safe 

otherwise might aswell not have the default case within updateDividentTax